### DISCUSSION TITLE
agent release for spark 3.4

### DISCUSSION THREAD

**Discussion by juja76** [AbsaOSS/spline-spark-agent#815] [Q&A]
hi there,

just wondering if there are any plans for the spline agent release for spark 3.4.

thank you!

**Comment by wajda** [comment]
there is a pull-request #793. It's on review/testing. I plan to merge it shortly after the testing is complete.

**Comment by juja76** [comment]
thank you very much! just to have an idea about the timeframe ... could it be ready anytime before the end of summer?

**Reply by wajda** [reply]
it's merged and the version released - 2.1.0. It's now being deployed to the Maven central. Should be available in repos somewhere during the day or so.
